servicemodules/discord.py

```diff
@@ -35,6 +36,7 @@ def __init__(self):
         self.utils = Utilities.get_instance()
         self.service_utils = self.utils
         self.modules = self.utils.modules_dict.values()
+        self.messages: Dict[int, Dict[str, Union[str, List[str]]]] = {}
```

Comment:
eeeh, what? line 209 has `self.messages[str(sent.id)] = ...,` which doesn't look like an `int` key ... why don't you see type errors, how does it work?

Reply:
My bad, I wrote that type before I knew for sure how I wanted to design it.  I decided to go with str instead of int because other services like Slack may use alphanumeric id's for their messages.  I'll fix that type annotation though.